lib/load_tester.rb

```diff
@@ -0,0 +1,21 @@
+class LoadTester
+  def trigger_delivery_request_workers(number)
+    email = create_test_email(to: success_address(0))
```

Comment:
should `success_address` be passed number?

Reply:
Yeah, I might have been thinking too much for the future here, but `success_address` returns a tagged email address with the argument passed in used as the tag.

Reply:
I'm more just questioning whether you meant to hardcode that always return "success+0@simulator.amazonses.com

Reply:
Oh, yeah, I figured for this test since it was fine since we only need to send to one address to get the load we need. For future tests, we'll need to have more variations.

Reply:
Oh yeah I understand more now, my brain was interpreting this as being within a loop.

Reply:
I couldn't decide whether to create an email for each delivery request worker, I figured in the end that we're only testing the worker, and that would be testing in the next level up.